src/java.base/share/classes/java/lang/ref/PhantomReference.java

```diff
@@ -45,6 +45,7 @@
  * phantom reference always returns {@code null}.
  * The {@link #refersTo(Object) refersTo} method can be used to test
  * whether some object is the referent of a phantom reference.
+ * @param<T> the type of the referent
```

Comment:
Shouldn't there be space after `@param` ?

Reply:
Good catch.  Sorry I missed it. This occurs in all `java/lang/ref` files.

Reply:
> Shouldn't there be a space after `@param` ?

> Good catch. Sorry I missed it. This occurs in all `java/lang/ref` files.

I built the API documentation after this PR has been integrated and the result was okay. I saw this output in every such case:
```
Type Parameters:
T - the type of the referent
```
javadoc is quite robust. However, for some IDEs such missing whitespace seems significant. Not only do they highlight the `@param` tag, but the type parameter information is missing from the rendered output.

Although it's not critical, we should fix it; I have filed JDK-8285890.

Reply:
> Good catch. Sorry I missed it. This occurs in all `java/lang/ref` files.

I've created a PR; feel free to review it at your convenience.